packages/web-components/src/global/variables.css

```diff
@@ -293,6 +293,7 @@
   /* z-index */
   --ic-z-index-base-value: 0;
   --ic-z-index-page-header: calc(var(--ic-z-index-base-value) + 10);
+  --ic-z-index-page-header-sticky: calc(var(--ic-z-index-base-value) + 120);
```

Comment:
minor one: could it be moved lower down, so that the they values are in ascending order?